README.md

```diff
@@ -11,7 +11,7 @@
 <p align="center">
   <img alt="Bundle Size" src="https://badgen.net/bundlephobia/minzip/@chakra-ui/react"/>
   <img alt="Language grade: JavaScript" src="https://img.shields.io/lgtm/grade/javascript/g/chakra-ui/chakra-ui.svg?logo=lgtm&logoWidth=18"/>
-  <img alt="MIT License" src="https://img.shields.io/github/license/chakra-ui/chakra-ui"/>
+  <img alt="MIT License" src="https://github.com/chakra-ui/chakra-ui/blob/main/LICENSE"/>
```

Comment:
Remove the image here is breaking the Image in the Readme.

Reply:
Sorry for that, I updated Readme once again thank you for the feedback.